convert2rhel/pkgmanager/__init__.py

```diff
@@ -58,3 +65,27 @@ def create_transaction_handler():
         from convert2rhel.pkgmanager.handlers.dnf import DnfTransactionHandler
 
         return DnfTransactionHandler()
+
+
+def clean_yum_metadata():
+    """Remove cached metadata from yum.
+
+    This is to make sure that Convert2RHEL works with up-to-date data from repositories before, for instance, querying
+    whether the system has the latest package versions installed, or before checking whether enabled repositories have
+    accessible URLs.
+    """
+    # We are using run_subprocess here as an alternative to call_yum_cmd
+    # which doesn't apply the correct --enablerepos option because at this point
+    # the tool didn't initialize the necessary functions in SystemInfo.
+    # The viable solution was calling the yum command as a subprocess manually
+    # instead of using that function wrapper.
+    output, ret_code = utils.run_subprocess(
+        ("yum", "clean", "metadata", "--enablerepo=*", "--quiet"), print_output=False
```

Comment:
All the other calls we have for run_subprocess are passing a list instead of a tuple. I think it should be good to follow the pattern there and change this one too. 

```suggestion
        ["yum", "clean", "metadata", "--enablerepo=*", "--quiet"], print_output=False
```